app/views/proposals/_encouragement_text.html.haml

```diff
@@ -1,7 +1,7 @@
 %p.lead
   - if @program.event_types.any?
     You can submit proposals for
-    = "#{event_types_sentence(@conference)}."
+    = "#{event_types_sentence(@conference, current_user&.is_admin || false)}."
```

Comment:
Yeah, i think this won't be necessary here.

Reply:
I was using the event_types_sentence to control which event types are displayed based on whether user is admin/non-admin. Should I just move that logic into the view code instead?